CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 - PR #2965 Fix slow orc reader perf with large uncompressed blocks
 
 ## Bug Fixes
-
+- PR #2981 enable build all targets without installation.
 
```

Comment:
There should be one full line between this as well. Also probably should be moved to the `0.12` section since this PR is re-targeted.

Reply:
moved to 0.12